_common

```diff
@@ -96,18 +96,19 @@ label_on_issue() {
     fi
 }
 
-snip_start="  # --- 8< ---"$'\n'
-snip_end="  # --- >8 ---"
+snip_start="    # --- 8< ---"$'\n'
+snip_end="    # --- >8 ---"
 handle_unknown() {
-    local testurl=$1 out=$2 reason=$3 group_id=$4 email_unreviewed=$5 from_email=$6 notification_address=${7:-}
-    local group_data group_name group_description group_mailto header excerpt=$snip_start
+    local testurl=$1 out=$2 reason=$3 group_id=$4 email_unreviewed=$5 from_email=$6 notification_address=${7:-} job_data=${8:-}
+    local group_data group_name group_description group_mailto header email excerpt=$snip_start
+    local job_name job_result job_info
     to_review+=("$testurl ${reason:0:50}")
-    header="$testurl : Unknown issue, to be reviewed -> $testurl/file/autoinst-log.txt"
-    echo "$header"
-    echo -e "Likely the error is within this log excerpt, last lines before shutdown:\n$snip_start"
+    header="[$testurl]($testurl): Unknown issue, to be reviewed"$'\n'"-> [autoinst-log.txt]($testurl/file/autoinst-log.txt)"$'\n'
```

Comment:
```suggestion
    job_name=$(echo "$job_data" | runjq -r '.job.name')
    header="[$job_name]($testurl): Unknown issue, to be reviewed"$'\n'"-> [autoinst-log.txt]($testurl/file/autoinst-log.txt)"$'\n'
```

Reply:
hm, the name is already written in the details, this would make the name redundant but the URL is also redundant in itself. For me it's ok as is regardless

Reply:
I don't care much about the link text, but I would like to keep the full URL visible somewhere even in the HTML form, because it includes the test number.
I don't want to have to hover over the link to see it.

Reply:
Good point.
Currently in the collapsed form the link at the beginning is swallowed completely:
![slack-html-email-collapsed](https://user-images.githubusercontent.com/688850/170707632-3e1d14ba-4545-4b7f-935e-68170d80e3a2.png)
which is unfortunate.
I will try out some things...

Reply:
Ok, if the link title is not a URL itself, it is displayed.
I think this is as good as it gets:

Collapsed:
![slack-html-email-collapsed-2](https://user-images.githubusercontent.com/688850/170730521-e1f790a0-d4d4-4787-9e6c-cc8b26331821.png)
Open:
![slack-html-email-2](https://user-images.githubusercontent.com/688850/170730527-dd1ba820-14fd-4026-b533-7f0e421b3b0f.png)

WDYT?

Reply:
Looks good Tina 😄